components/org.wso2.carbon.identity.oauth/pom.xml

```diff
@@ -310,6 +310,21 @@
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.identity.central.log.mgt</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.13.4</version>
```

Comment:
All the dependencies should be added to the dependency management section of the parent pom and the versions should be defined as a property in the parent pom.